consultation_analyser/consultations/views/answers.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth.decorators import login_required
+from django.core.paginator import Paginator
+from django.http import HttpRequest
+from django.shortcuts import get_object_or_404, render
+
+from .. import models
+from .decorators import user_can_see_consultation
+
+
+@user_can_see_consultation
+@login_required
+def index(
+    request: HttpRequest,
+    consultation_slug: str,
+    question_slug: str,
+):
+    # For now, just display free text parts of the question
+    consultation = get_object_or_404(models.Consultation, slug=consultation_slug)
+
+    question = models.Question.objects.get(
+        slug=question_slug,
+        consultation=consultation,
+    )
+    # if processing_run:
```

Comment:
Do we need to keep this? Have we got rid of processing runs?

Reply:
Good spot - yes we have. Will remove.

Reply:
Now deleted.